network.go

```diff
@@ -52,6 +52,8 @@ type NetworkController interface {
 	// Create a new network. The options parameter carry driver specific options.
 	// Labels support will be added in the near future.
 	NewNetwork(d *NetworkDriver, name string, options interface{}) (Network, error)
+	// Register an external driver.
+	RegisterDriver(name string, driver driverapi.Driver)
```

Comment:
@tomwilkie As you know we are all currently working with Docker plugin backend designers. Based on what comes out of it, the driver integration/registration might change. Also, I dont think it is a good idea to expose the driverapi to the Northbound API towards the applications using Libnetwork.
That is the reason we keep the internalDriver non-exposed outside the package. Just because of the fact that this is still WIP.

Hence I think we should wait for the plugin design to settle down so that we do the changes not just here. But also in other APIs. Hope that make sense.